torch_ort_inference/docker/Dockerfile.ort-infer-stable-torch1120-onnxruntime1120-openvino-ubuntu18.04

```diff
@@ -0,0 +1,85 @@
+#-------------------------------------------------------------------------
+# Copyright(C) 2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
+#--------------------------------------------------------------------------
+
+ARG OPENVINO_VERSION="2022.1.0"
+ARG OVPT_BRANCH="releases/v0.5"
```

Comment:
ARG "OVPT_BRANCH" name should be changed to "TORCHORT_BRANCH" and "releases/v0.5" should be "main".

Reply:
fixed